helm/templates/deployment.yaml

```diff
@@ -35,17 +35,17 @@ spec:
             - /manager
           env:
             - name: RELATED_IMAGE_EBPF_AGENT
-              value: quay.io/netobserv/netobserv-ebpf-agent:v1.9.0-community
+              value: '{{ .Values.ebpfAgent.image }}:{{ .Values.ebpfAgent.version }}'
             - name: RELATED_IMAGE_FLOWLOGS_PIPELINE
-              value: quay.io/netobserv/flowlogs-pipeline:v1.9.0-community
+              value: '{{ .Values.flowlogsPipeline.image }}:{{ .Values.flowlogsPipeline.version }}'
             - name: RELATED_IMAGE_CONSOLE_PLUGIN
-              value: '{{ if .Values.standaloneConsole.enable }}quay.io/netobserv/network-observability-standalone-frontend:v1.9.0-community{{ else }}quay.io/netobserv/network-observability-console-plugin:v1.9.0-community{{ end }}'
+              value: '{{ if .Values.standaloneConsole.enable }}{{ .Values.standaloneConsole.image }}:{{ .Values.standaloneConsole.version }}{{ else }}{{ .Values.consolePlugin.image }}:{{ .Values.consolePlugin.version }}{{ end }}'
             - name: RELATED_IMAGE_CONSOLE_PLUGIN_COMPAT
-              value: quay.io/netobserv/network-observability-console-plugin-pf4:v1.8.2-community
+              value: '{{ if .Values.standaloneConsole.enable }}{{ .Values.standaloneConsole.image }}:{{ .Values.standaloneConsole.version }}{{ else }}{{ .Values.consolePlugin.image }}:{{ .Values.consolePlugin.version }}{{ end }}'
```

Comment:
Is it on purepose to use the same image / version for the compat console plugin and the current one ?

If so, maybe we could simply make `RELATED_IMAGE_CONSOLE_PLUGIN_COMPAT` env optionnal ?

Reply:
yeah, being a bit lazy here, the few people (if any) who'd need compat could just switch manually to the correct image; let me see to make it optional then